### PR TITLE
tidb/config: Add two configuration

### PR DESCRIPTION
**Pull request by ming-relax**
Extract the configuration as a package and add two configurations for tidb
    - SlowThreshold
    - QueryLogMaxlen

### CODE REVIEW

**Comment by tiancaiamao** [comment]
CI fail. @ming-relax

**Comment by ming-relax** [comment]
@tiancaiamao fixed CI, can you take a look?

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 PingCAP, Inc.
```
Comment (tiancaiamao) [inline config/config.go]: s/2015/2017

**Comment by tiancaiamao** [comment]
LGTM
PTAL @zimulala @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**